src/libsyncengine/syncpal/syncpal.cpp

```diff
@@ -1052,6 +1061,8 @@ ExitCode SyncPal::fileRemoteIdFromLocalPath(const SyncPath &path, NodeId &nodeId
 bool SyncPal::checkIfExistsOnServer(const SyncPath &path, bool &exists) const {
     exists = false;
 
+    if (!_remoteSnapshot) return false;
```

Comment:
Can this be unit tested in a simple way?

Reply:
Done